packages/core/src/common/console-logger-server.ts

```diff
@@ -7,25 +7,42 @@
 
 import { injectable } from "inversify";
 import { LogLevel } from './logger';
-import { ILoggerServer, ILoggerClient } from './logger-protocol';
+import { ILoggerServer } from './logger-protocol';
+
+const originalConsoleLog = console.log;
 
 @injectable()
 export class ConsoleLoggerServer implements ILoggerServer {
-    setLogLevel(name: string, logLevel: number): Promise<void> {
-        return Promise.resolve();
+
+    async setLogLevel(): Promise<void> {
+        // NOOP
     }
-    getLogLevel(name: string): Promise<number> {
-        return Promise.resolve(LogLevel.DEBUG);
+
+    async getLogLevel(): Promise<number> {
+        return LogLevel.INFO;
     }
-    log(name: string, logLevel: number, message: string, params: any[]): Promise<void> {
-        console.log(`${message} ${params.map(p => p.toString()).join(' ')}`);
-        return Promise.resolve();
+
+    // tslint:disable-next-line:no-any
+    async log(name: string, logLevel: number, message: string, params: any[]): Promise<void> {
+        const severity = `${(LogLevel.strings.get(logLevel) || 'info').toUpperCase()}`;
+        const dateTime = this.formatDate();
+        originalConsoleLog(`[${dateTime}] ${name} ${severity}: ${message} ${params.map(p => p.toString()).join(' ')}`);
```

Comment:
Should we log severity error and fatal to `log.error`?

Reply:
I am also not sure if we should add the data. Frameworks like stackdriver would add them again.
Also, we should leave the serialization to console.log/error. I.e. simply calling `originalConsoleLog(`${name} ${severity}`, message, ...params)`

Reply:
> Should we log severity error and fatal to log.error?

I do not mind doing that.

> I am also not sure if we should add the data. Frameworks like stackdriver would add them again.

I do not understand this. Please explain.

> Also, we should leave the serialization to console.log/error. I.e. simply calling originalConsoleLog(${name} ${severity}, message, ...params)

OK. It looks strange without the date-time, but I do not have any objections. Let's do that.